src/jlgen.jl

```diff
@@ -32,6 +32,7 @@ function unsafe_function_from_type(ft::Type)
         Ref{ft}()[]
     end
 end
+global MethodError
```

Comment:
```suggestion
function MethodError end
```

imo, it's more conspicuous that this does what you want

Reply:
Actually that looks more confusing since it seems redundant to the statement below.

Reply:
Fair enough - I agree that neither syntax is really very transparent / consistent